README.md

```diff
@@ -36,6 +36,11 @@ Usage of marathon_exporter:
   -marathon.uri string
         URI of Marathon (default "http://marathon.mesos:8080")
         Note: Supply HTTP Basic Auth (i.e. user:password@example.com)
+	            If you URI inclube '%' ,you should used marathon.username and marathon.password
+  -marathon.username string
+	marathon author username ,(default "")
+  -marathon.password string
+        marathon author password ,(default "")
```

Comment:
Please remove both unnecessary ','
Also, change 'author' -> 'user' to be clearer for the user.

Reply:
I'm fix it.